lib/rucio/core/transfer.py

```diff
@@ -188,6 +188,82 @@ def set_transfers_state(transfers, submitted_at, session=None):
         raise RucioException(error.args)
 
 
+@read_session
+def get_next_transfers_new(request_type, state, limit=100, older_than=None, rse=None, activity=None,
```

Comment:
For me the code looks good, please just add some kind of TODO comment, both at the get_next_transfers_new and get_next_transfers that one of them will be deprecated and replaced by the other.